src/components/MediaPlayerCarousel.tsx

```diff
@@ -200,19 +203,22 @@ export class MediaPlayerCarousel extends React.PureComponent<Props, State> {
               onLayout={(event) => {
                 this.setState({ explosionOrigin: event.nativeEvent.layout.y })
               }}
-              disabled={boostPaymentLoading}
+              disabled={boostWasSent}
               style={styles.boostButton}
               onPress={this._attemptBoost}>
-              {boostPaymentLoading ? (
-                <ActivityIndicator />
-              ) : (
-                <>
-                  <Text testID='boost_button_text_1'>{translate('Boost').toUpperCase()}</Text>
-                  <Text testID='Boost Button_text_2' style={{ fontSize: PV.Fonts.sizes.xs }}>
-                    {boostAmount} {translate('sats')}
-                  </Text>
-                </>
-              )}
+                {boostIsSending ? 
+                  <ActivityIndicator />
+                  :
+                  <>
+                    <Text testID='boost_button_text_1'>{boostText}</Text>
+                    {
+                      !boostWasSent &&
```

Comment:
Is this only going to show the amount of the boost when the boost was not sent?

Reply:
Yeah, I guess that's unnecessary. I'll remove it.